contrib/NeonVM/util.h

```diff
@@ -0,0 +1,15 @@
+#ifndef __UTIL_H
+#define __UTIL_H
+
+void exec_error(const char *msg, ...);
+void fatal_error(const char *msg, ...);
+
+#ifndef BOOL

```

Comment:
This #ifndef check doesn't test for whether BOOL has been typedef'd before. All it checks is whether there has been a #define BOOL (preprocessor only) before. The preprocessor cannot see typedefs. Remove this #ifndef because it's completely unnecessary.

Reply:
Removed.